numpy/core/defchararray.py

```diff
@@ -1785,6 +1785,26 @@ def isnumeric(a):
     --------
     unicode.isnumeric
 
+    Examples
+    --------
+    >>> np.char.isnumeric('¼')
+    array(True)
+
+    >>> np.char.isnumeric('123')
+    array(True)
+
+    >>> np.char.isnumeric('abc')
+    array(False)
+
+    >>> np.char.isnumeric('Ⅷ')
+    array(True)
+
```

Comment:
I'm -1 on the scalar examples. Users shouldn't be using `np.char.isnumeric` to evaluate single strings, they should be using the `str.isnumeric` method directly. Therefore this should be trimmed down to the array input example only:

```suggestion
```

Reply:
Removed!